src/components/LessonSlider.tsx

```diff
@@ -79,6 +79,11 @@ const LessonSlider: React.FC<{
                 lessonData={lessonData}
                 startIndex={startIndex === -1 ? startIndex + 1 : startIndex}
               />
+              {m.chapterTitle && 
```

Comment:
set this UI on lessonCard

Reply:
![localhost_3000_home(iPhone XR)](https://github.com/chimple/cuba/assets/134476789/7a77af68-7b1e-4490-9edb-04bb89edf3f3)